c-pallets/file-map/src/lib.rs

```diff
@@ -197,6 +261,7 @@ pub mod pallet {
 		#[pallet::weight(10_000)]
 		pub fn init_public_key(origin: OriginFor<T>) -> DispatchResult {
 			let _ = ensure_root(origin)?;
+			// why this hardcoded values?
```

Comment:
In fact, this is a public key related to our certification, which is only a temporary practice. After we use the new certification library, we will not do this again